scripts/node/README.md

```diff
@@ -0,0 +1,37 @@
+# Steps to benchmark node scanning 
+
+## Pre steps:
+
+1.Make sure docker is installed on your local laptop
+2.execute
+```
+docker run -it --rm registry.access.redhat.com/ubi8/ubi
+```
+3.make sure **Vim, RPM and RPM-build** are installed in the running ubi8 bash. If not, do yum search <package name> and yum install <package name>
+
+---------------------------------------------------------------------------------------------
+
+(optional) create a temporary/target folder in the running ubi8 bash for RPM specs. e.g mkdir temp-specs
+1.go to the target folder and create a python file to generate RPM specs (see the generate-junit-reports.sh in scanner/scripts folder)
```

Comment:
How about we mount the host folder that contains this script instead of instructing the user to create it?
Also, is it a python file or are you talking about `generate-rpm-specs.sh` here?